modules/packages/FFTW.chpl

```diff
@@ -252,8 +252,8 @@ module FFTW {
     param rank = input.rank;
 
     var dims: c_array(c_int,rank);
-    for param i in 1..rank do
-      dims(i-1) = input.domain.dim(i).size.safeCast(c_int);
+    for param i in 0..rank-1 do
```

Comment:
Is there a reason to use `0..#rank` above and `0..rank-1` here? Can we unify on one or the other? Should we instead of use `0..<rank` (I'm not sure what the new preferred style is)

Reply:
`0..#rank` unfortunately doesn't support `param` unrolling at present (issue #15250).  `0..<rank` does, so we could use that form more consistently in this module (it just wasn't implemented when I originally made most of the changes in this PR).

Reply:
https://github.com/chapel-lang/chapel/pull/14522/commits/6bd92c917e72ddb2ec633160c6aed667bce759a7 implements these ranges as `0..<` style ranges for consistency (unfortunately, the use of `rank-1` upper bounds in some of the loops makes things a bit confused even with that unification...  hopefully I didn't introduce any errors in either of these sets of edits).